src/main/java/gregtech/api/fluids/MetaFluids.java

```diff
@@ -0,0 +1,253 @@
+        setMaterialFluidTexture(Materials.HydrogenSulfide, FluidTypes.GAS);
+        setMaterialFluidTexture(Materials.SulfuricGas, FluidTypes.GAS);
+        setMaterialFluidTexture(Materials.RefineryGas, FluidTypes.GAS);
+        setMaterialFluidTexture(Materials.SulfuricNaphtha, FluidTypes.LIQUID);
+        setMaterialFluidTexture(Materials.SulfuricLightFuel, FluidTypes.LIQUID);
+        setMaterialFluidTexture(Materials.SulfuricHeavyFuel, FluidTypes.LIQUID);
+        setMaterialFluidTexture(Materials.Naphtha, FluidTypes.LIQUID);
+        setMaterialFluidTexture(Materials.LightFuel, FluidTypes.LIQUID);
+        setMaterialFluidTexture(Materials.HeavyFuel, FluidTypes.LIQUID);
+        setMaterialFluidTexture(Materials.LPG, FluidTypes.GAS);
+        setMaterialFluidTexture(Materials.LightlySteamCrackedLightFuel, FluidTypes.LIQUID);
+        setMaterialFluidTexture(Materials.SeverelySteamCrackedLightFuel, FluidTypes.LIQUID);
+        setMaterialFluidTexture(Materials.LightlySteamCrackedHeavyFuel, FluidTypes.LIQUID);
+        setMaterialFluidTexture(Materials.SeverelySteamCrackedHeavyFuel, FluidTypes.LIQUID);
+        setMaterialFluidTexture(Materials.Chlorine, FluidTypes.GAS);
+        setMaterialFluidTexture(Materials.CetaneBoostedDiesel, FluidTypes.LIQUID);
+        setMaterialFluidTexture(Materials.SodiumPersulfate, FluidTypes.LIQUID);
+        setMaterialFluidTexture(Materials.GlycerylTrinitrate, FluidTypes.LIQUID);
+        setMaterialFluidTexture(Materials.Lubricant, FluidTypes.LIQUID);
+        setMaterialFluidTexture(Materials.Creosote, FluidTypes.LIQUID);
+        setMaterialFluidTexture(Materials.SeedOil, FluidTypes.LIQUID);
+        setMaterialFluidTexture(Materials.Oil, FluidTypes.LIQUID);
+        setMaterialFluidTexture(Materials.Diesel, FluidTypes.LIQUID);
+        setMaterialFluidTexture(Materials.Biomass, FluidTypes.LIQUID);
+        setMaterialFluidTexture(Materials.Ethanol, FluidTypes.LIQUID);
+        setMaterialFluidTexture(Materials.SulfuricAcid, FluidTypes.ACID);
+        setMaterialFluidTexture(Materials.Milk, FluidTypes.LIQUID);
+        setMaterialFluidTexture(Materials.McGuffium239, FluidTypes.LIQUID);
+        setMaterialFluidTexture(Materials.Glue, FluidTypes.LIQUID);
+        setMaterialFluidTexture(Materials.HydrochloricAcid, FluidTypes.ACID);
+        setMaterialFluidTexture(Materials.LeadZincSolution, FluidTypes.LIQUID);
+        setMaterialFluidTexture(Materials.NaturalGas, FluidTypes.GAS);
+        setMaterialFluidTexture(Materials.Blaze, FluidTypes.LIQUID);
+        setMaterialFluidTexture(Materials.FluoroantimonicAcid, FluidTypes.ACID);
+        setMaterialFluidTexture(Materials.Naquadah, FluidTypes.LIQUID);
+        setMaterialFluidTexture(Materials.NaquadahEnriched, FluidTypes.LIQUID);
+        setMaterialFluidTexture(Materials.Naquadria, FluidTypes.LIQUID);
+        setMaterialFluidTexture(Materials.Ice, FluidTypes.LIQUID);
+        setMaterialFluidTexture(Materials.UUMatter, FluidTypes.LIQUID);
```

Comment:
Can those not be set on fluid registration automatically? Or are they different?

Reply:
These are used to specify custom texture locations for specific fluids. It is better that these are done manually so that we don't need to check for files existing in directories every time we generate a fluid.